packages/shared/src/components/badges/TopReaderBadge.tsx

```diff
@@ -0,0 +1,128 @@
+    />
+    <defs>
+      <linearGradient
+        id="goldGradient"
+        x1="0"
+        y1="0"
+        x2="79.5564"
+        y2="71.6044"
+        gradientUnits="userSpaceOnUse"
+      >
+        <stop stopColor={colors.bun['10']} />
+        <stop offset="0.275" stopColor={colors.cheese['10']} />
+        <stop offset="0.5" stopColor={colors.bun['10']} />
+        <stop offset="0.75" stopColor={colors.cheese['10']} />
+        <stop offset="1" stopColor={colors.bun['10']} />
+      </linearGradient>
+
+      {/* This is a hack so that we can properly center the image without fiddling with html and css */}
+      <pattern
+        id="profileImage"
+        preserveAspectRatio="xMidYMid slice"
+        x="0"
+        y="0"
+        width="1"
+        height="1"
+        viewBox="0 0 512 512"
+      >
+        <image width="512" height="512" href={imageUrl} />
+      </pattern>
+    </defs>
+  </svg>
+);
+
+export const TopReaderBadge = ({
+  user,
+  topReader,
+}: {
+  user: LoggedUser;
+  topReader: {
+    date: string;
+    keyword: Pick<Keyword, 'flags'>;
+  };
+}): JSX.Element => {
+  const date = new Date(topReader.date);
+  const issuedAt = date.toLocaleString('en-US', {
+    year: 'numeric',
+    month: 'long',
+  });
+  return (
+    <div
+      className="invert flex h-80 w-80 rounded-24 p-1 text-center text-text-primary"
+      style={{
+        backgroundImage: themeToLinearGradient[DevCardTheme.Gold],
+      }}
+    >
+      {/* @TODO: use correct image */}
+      <div className="flex max-w-full flex-1 flex-col items-center rounded-20 bg-black bg-[url('https://i.bratteng.com/1730314661865_xhfx89.png')] bg-cover px-3 py-5">
+        <div className="flex justify-center pb-1">
+          <BadgeIcon imageUrl={user.image} />
+        </div>
+
+        <p className="font-bold typo-footnote">{user.name}</p>
+
+        <p className="pb-1 text-text-secondary typo-caption1">
+          @{user.username}
+        </p>
+
+        <h1 className="pb-1 font-bold typo-title2">Top reader</h1>
+
+        <p className="text-text-secondary typo-caption1">{issuedAt}</p>
+
+        <div
+          className="relative my-5 max-w-max rounded-12 p-1"
+          style={{
+            backgroundImage: themeToLinearGradient[DevCardTheme.Gold],
+          }}
+        >
+          <div className="relative overflow-hidden rounded-8 px-2 py-0.5">
+            <span className="relative z-1 font-bold text-black typo-title2">
+              {topReader.keyword.flags?.title}
+            </span>
+            <div
+              className="absolute left-0 top-0 z-0 h-full w-full -scale-x-100"
+              style={{
+                backgroundImage: themeToLinearGradient[DevCardTheme.Gold],
+              }}
+            />
+          </div>
+        </div>
+        <div className="flex">
+          <LogoIcon className={{ container: 'h-logo' }} />
+          <LogoText className={{ container: 'ml-1 h-logo' }} />
+        </div>
```

Comment:
Looked at using Logo component, but it adds a link to it, and I just need the actual logo.

Reply:
This is fine, we could potentially combine these 2 into 1 file, and re-use that inside logo and here.
But don't think it's worth it.